sdk/communication/Azure.Communication.CallingServer/src/CallingServerClientOptions.cs

```diff
@@ -41,7 +42,13 @@ public enum ServiceVersion
             /// The Beta of the CallingServer service.
             /// </summary>
 #pragma warning disable CA1707 // Identifiers should not contain underscores
-            V2021_06_15_Preview = 1
+            V2021_06_15_Preview = 1,
+#pragma warning restore CA1707 // Identifiers should not contain underscores
+            /// <summary>
+            /// The Beta of the CallingServer service.
```

Comment:
Better to have distinct comments for the 2 versions.
Something like 'The 2021_08_30 version of the CallingServer service.' And similarly for the one above.